src/components/Header/NavBar.tsx

```diff
@@ -24,6 +26,42 @@ export const NavBar = ({ path, showSearch = true }: { path: string; showSearch?:
     }
   }
 
+  React.useEffect(() => {
+    if (
+      !window.location.hostname.includes("localhost") &&
+      !window.location.hostname.includes("documentation-private-git-")
```

Comment:
Does this mean it's only meant to have have the language bar in prod?
Is there an env var available that allows us not to leak our preview deployment domain?

Reply:
> Does this mean it's only meant to have have the language bar in prod?
Correct, it's domain dependent.

> Is there an env var available that allows us not to leak our preview deployment domain?
No. We can protect them or remove them, however, since docs are oss it should not be an issue.